docs/FrameworkControllerMode.md

```diff
@@ -0,0 +1,84 @@
+**Run an Experiment on FrameworkController**
+===
+NNI supports running experiment using [FrameworkController](https://github.com/Microsoft/frameworkcontroller), called frameworkcontroller mode. FrameworkController is built to orchestrate all kinds of applications on Kubernetes, you could use frameworkcontroller as a training service to run your experiment.
```

Comment:
Now you can use frameworkcontroller as the training service to run NNI experiment.